layouts/partials/head.html

```diff
@@ -1,7 +1,7 @@
 <head>
     <meta charset="UTF-8">
     <meta http-equiv="X-UA-Compatible" content="IE=edge,chrome=1">
-    <meta name="viewport" content="width=device-width, initial-scale=1.0, maximum-scale=1.0">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0, maximum-scale=1.0"> 
```

Comment:
Little thing -- you might want to set your editor to trim trailing whitespace and add empty lines. If you're using VSCode, the settings (in settings.json) should be:

<img width="382" alt="image" src="https://user-images.githubusercontent.com/274700/107442166-7358f280-6aeb-11eb-8383-54a228cca98c.png">